package.json

```diff
@@ -14,7 +14,7 @@
   "license": "./LICENSE",
   "scripts": {
     "start": "node server.js",
-    "start:dev": "NODE_ENV=development hof-build watch",
+    "start:dev": "NODE_ENV=development hof-build watch --env",
```

Comment:
I'd remove this from `start:dev`
that's the point of the `dev` script below

Reply:
I have removed the start:dev script now